src/bundles/SellerRegistration/redux/modules/reducer.js

```diff
@@ -9,16 +9,22 @@ import signup from './signup';
 import application from './application';
 import errorMessage from '../../../../shared/reduxModules/errorMessage';
 import form_options from '../../../../shared/reduxModules/form_options';
+import application_errors from './application_errors';
+import agreement from './agreement'
+import domains from './domains'
 
 export default combineReducers({
+  agreement,
   application,
+  application_errors,
```

Comment:
Should we use camel case?

Reply:
annoyingly, this needs to match the response